driver-db/ens160.toml

```diff
@@ -0,0 +1,34 @@
+manifest_version = "0.0.1"
+
+[meta]
+categories = ["Sensor::TVOC", "Sensor::eCO2"]
+datasheets = [
+    "https://www.sciosense.com/wp-content/uploads/2023/12/ENS160-Datasheet.pdf",
+]
+manufacturer = "ScioSense"
+names = ["ENS160"]
+part_numbers = ["ENS160"]
+
+kicad_symbol = ["ENS160-BGLM"]
```

Comment:
I can not find this part in my KiCAD installation. I also think it is missing the library name?

Reply:
I never used KiCAD, maybe you can find the part for the driver?

Reply:
Oh, maybe there just isn't a part available for it, so you can just leave the list empty